jib-core/src/main/java/com/google/cloud/tools/jib/configuration/BuildContext.java

```diff
@@ -532,7 +532,21 @@ public boolean getAlwaysCacheBaseImage() {
    * @return a new {@link RegistryClient.Factory}
    */
   public RegistryClient.Factory newBaseImageRegistryClientFactory() {
-    return newRegistryClientFactory(baseImageConfiguration);
+    return newBaseImageRegistryClientFactory(baseImageConfiguration.getImageRegistry());
```

Comment:
should this javadoc now indicate that the serverUrl is derived from the baseimage configuration?